### PR TITLE
[DOCS] Tutorial for LiveServer usage

### PR DESCRIPTION
**Pull request by TheCedarPrince**
This introduces a tutorial to complement #164 based on a long work session with @asinghvi17 .

@asinghvi17 , for deployments, will the creator need to remove these tweaks from their `make.jl` file? Or can these tweaks be safely left in when deployed? I wasn't sure if they could be left in or not.

Otherwise, feel free to make additional tweaks here as needed!

### CODE REVIEW

**Comment by asinghvi17** [comment]
For deployment you would need to remove all tweaks, maybe we can make a convenience function to do this for you though (something like `MarkdownVitepress(...) |> live_serveable` maybe.

**Comment by TheCedarPrince** [comment]
That sounds great @asinghvi17 -- do we want to make that functionality now and then merge these two PRs or go ahead and merge the tutorial + your functionality that you just added and make this `live_serveable` function later? :thinking:

File: docs/src/getting_started.md [hunk 1]
```diff
@@ -14,9 +14,40 @@ and that should be it!
...
+
+  ii. `build_vitepress = false`
+
+3. Within the `makedocs` command within your `make.jl` file, pass the following key word argument:
+
+  i. `clean = false`
+
+Then, to preview changes live, open two separate Julia instances both within the `docs` folder and activate the `docs` environment in both sessions.
+Within one session run `using LiveServer; servedocs(foldername="/path/to/your/docs/folder")` (this could be something like `servedocs(foldername="docs/")` or `servedocs(foldername=pwd())`).
+In the other session run `DocumenterVitepress.dev_docs("build", md_out_path = "")`.
```
Comment (asinghvi17) [inline docs/src/getting_started.md:36]: Md_output_path

File: docs/src/getting_started.md [hunk 2]
```diff
@@ -14,9 +14,40 @@ and that should be it!
 
 The section [Advanced method](@ref) describes how to get more control over your Vitepress build.
 
-### Developing docs locally
+### Preview Documentation Development Instantly
 
-In order to develop docs locally, you can add the `clean=false` parameter to `makedocs`, and call `DocumenterVitepress.dev_docs("/path/to/docs/build")` in some other REPL.  This works with LiveServer.jl based approaches as well.  Note that the `dev_docs` call cannot be in the file that LiveServer is watching
+You can preview your documentation development changes locally, instantly by adding [LiveServer.jl](https://github.com/tlienart/LiveServer.jl) into your `docs` environment and making a few tweaks to your `make.jl` configuration.
+Here are the tweaks to add:
```
Comment (asinghvi17) [inline docs/src/getting_started.md:20]: (and remove when you finalize your commit)

**Comment by TheCedarPrince** [comment]
Added this in @asinghvi17 !

**Review by asinghvi17 — APPROVED** [review]
Thanks @thecedarprince!

**Comment by TheCedarPrince** [comment]
Right back at ya @asinghvi17

**Pull request merged**